src/lava/magma/core/model/py/ports.py

```diff
@@ -66,14 +73,14 @@ class PyInPortVectorDense(PyInPort):
     def recv(self) -> np.ndarray:
         return ft.reduce(
             lambda acc, csp_port: acc + csp_port.recv(),
```

Comment:
Is this a potential performance bottleneck? Should we call reduce when there is only one port?

Is functools reduce the best thing to use rather than a pure numpy function? Not sure if it uses that under the hood.

Reply:
Not sure, but this is not the scope of this PR and was not changed.